Well-Coffee-UI/src/components/pages/LoginPage.jsx

```diff
@@ -1,10 +1,43 @@
 import React from 'react'
+import { useState } from 'react';
+import axios from 'axios';
 
 /* 
   Created by Dominique Gould 
 */
 
-const LoginPage = () => {
+function LoginPage({ setAuthenticated }) {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [message, setMessage] = useState("");
+
+  // const BASEAPIURL = "http://localhost:8080/api/login";
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+
+    console.log("Is it working?");
+    console.log(email, password);
+    try {
+      const response = await axios.post("http://localhost:8080/api/login", 
+      {
+        email, 
+        password
+      }, 
+      {
+        withCredentials: true,
```

Comment:
Does the login call need to be sent with credentials?  The credentials don't exist yet, right?